.circleci/config.yml

```diff
@@ -133,7 +133,11 @@ jobs:
           name: Execute integration test suites
           working_directory: test/integration/
           command: |
-            circleci tests glob "suites/*" | circleci tests split | xargs ./test.sh
+            # CircleCI is silly and doesn't provide this incredibly helpful
+            # environment variable. Requests for it go back years. For shame.
+            CIRCLE_PR_BRANCH=$(curl -s https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/pulls/${CIRCLE_PR_NUMBER} | jq -r '.base.ref')
+            echo "CIRCLE_PR_BRANCH: ${CIRCLE_PR_BRANCH}"
```

Comment:
Is this `echo` useful in CI logs or just here for debugging?

Reply:
I didn't see the actual value in the logs, so the echo made it easy to ensure the right thing was being passed. That being said, I'm ok getting rid of it.